sentry-kotlin-multiplatform/src/appleMain/kotlin/io/sentry/kotlin/multiplatform/SentryBreadcrumb.kt

```diff
@@ -0,0 +1,106 @@
+package io.sentry.kotlin.multiplatform
+
+import io.sentry.kotlin.multiplatform.extensions.toCocoaSentryLevel
+import io.sentry.kotlin.multiplatform.extensions.toKMPSentryLevel
+import cocoapods.Sentry.SentryBreadcrumb as CocoaBreadcrumb
+
+actual class SentryBreadcrumb {
```

Comment:
That feels like a lot of duplication compared to the other one, the only difference here is that it talks to `CocoaBreadcrumb`.
Maybe we can think a way to generify this.